README.md

```diff
@@ -1,4 +1,13 @@
 Docker Toolbox
+
+### TL/DR: Docker will be ceasing Support and Development of the Docker Toolbox project in 2020 - going forward please use [Docker Desktop](https://www.docker.com/products/docker-desktop)
```

Comment:
Maybe add some relevant emoji to draw attention ⚠️ or ❗ or 🔴 or 🛑

Reply:
Added